snapcraft/plugins/nodejs.py

```diff
@@ -31,6 +31,9 @@
     - node-engine:
       (string)
       The version of nodejs you want the snap to run on.
+    - npm-run:
+      (list)
+      A list of targets to `npm run`
```

Comment:
Would it be useful to explain here that the run tasks are executed after npm install? I'm a n00b, so I don't know if that's obvious.